src/c++/vdo/base/data-vio.c

```diff
@@ -1422,7 +1422,7 @@ void handle_data_vio_error(struct vdo_completion *completion)
  */
 const char *get_data_vio_operation_name(struct data_vio *data_vio)
 {
-	STATIC_ASSERT((MAX_VIO_ASYNC_OPERATION_NUMBER - MIN_VIO_ASYNC_OPERATION_NUMBER) ==
```

Comment:
Here, too, the indentation on the next line needs to be fixed (moved one space to the left).